File: src/priv/bajdcc/LALR1/interpret/os/user/routine/URShell.java
package priv.bajdcc.LALR1.interpret.os.user.routine;

import priv.bajdcc.LALR1.interpret.os.IOSCodePage;
import priv.bajdcc.util.ResourceLoader;

/**
 * 【用户态】脚本解释器
 *
 * @author bajdcc
 */
public class URShell implements IOSCodePage {
	@Override
	public String getName() {
		return "/usr/p/sh";
	}

	@Override
	public String getCode() {
		return ResourceLoader.load(getClass());
	}
}

File: src/main/kotlin/com/bajdcc/LALR1/interpret/os/user/routine/URShell.kt
package com.bajdcc.LALR1.interpret.os.user.routine

import com.bajdcc.LALR1.interpret.os.IOSCodePage
import com.bajdcc.util.ResourceLoader

/**
 * 【用户态】脚本解释器
 *
 * @author bajdcc
 */
class URShell : IOSCodePage {
    override val name: String
        get() = "/usr/p/sh"

    override val code: String
        get() = ResourceLoader.load(javaClass)
}
